include/odp/api/spec/ipsec.h

```diff
@@ -1626,6 +1696,18 @@ int odp_ipsec_sa_mtu_update(odp_ipsec_sa_t sa, uint32_t mtu);
  */
 void *odp_ipsec_sa_context(odp_ipsec_sa_t sa);
 
+/**
+ * Retrieve information about an IPSEC SA
+ *
+ * @param      sa       The IPSEC SA for which to retrieve information
+ * @param[out] sa_info  Pointer to the structure which stores the IPSEC SA
```

Comment:
This wording makes it a bit unclear who allocates and owns the memory pointed to by sa_info. How about "pointer to caller allocated SA info structure to be filled in"?

Reply:
Agreed. Will update it to "Pointer to caller allocated SA info structure to be filled in".